src/test/java/net/torocraft/minecoprocessors/processor/ProcessorTest.java

```diff
@@ -1,884 +0,0 @@
-package net.torocraft.minecoprocessors.processor;
```

Comment:
Where these tests causing trouble for the upgrade?  I would like to keep them as they help catch a few bugs that would have been hard to track down other wise.

Reply:
Actually I have to double check, I commented out a lot of things to get an initial testable version with blocks, items, and recipes. The way things are done have changed quite a lot between 1.12 and 1.13.
Oh I just see there was a test directory in the original code! Let's re-add this.

Reply:
This looks like it is going in the right direction to me.  I am going to merge to keep things going.  If I get time tonight I will see if I can get the new code running.  Thanks for taking this on!